src/Core/Managed/Shared/TelemetryClient.cs

```diff
@@ -44,6 +45,12 @@ public TelemetryClient(TelemetryConfiguration configuration)
             }
 
             this.configuration = configuration;
+
+            // If someone created configuration from scratch and forgot to initialize the channel then use the default.
```

Comment:
This comment and code doesn't agree with each other. From the comment what i understand is if channel is null, default is used, but in code we throw Ex if channel is null.